bundles/org.openhab.core.compat1x.test/.classpath

```diff
@@ -4,5 +4,6 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/test/java"/>
 	<classpathentry kind="src" path="src/test/groovy"/>
+	<classpathentry exported="true" kind="con" path="GROOVY_DSL_SUPPORT"/>
```

Comment:
Should this be added?

Reply:
Hm, no. This has been added by the Eclipse IDE.
I should use
```
git add '**/MANIFEST.MF'
```
but haven't use it. :cry: